docs/content/doc/installation/from-source.en-us.md

```diff
@@ -145,7 +145,7 @@ launched manually from command line, it can be killed by pressing `Ctrl + C`.
 
 Gitea will search for a number of things from the _`CustomPath`_. By default this is
 the `custom/` directory in the current working directory when running Gitea. It will also
-look for its configuration file _`CustomConf`_ in _`CustomPath`_/conf/app.ini`, and will use the
+look for its configuration file _`CustomConf`_ in _`CustomPath_/conf/app.ini`, and will use the
```

Comment:
### Preview

Proposed: 
_`CustomPath_/conf/app.ini`

delvh 1:
_`CustomPath`_/conf/app.ini

delvh 2:
_`CustomPath/conf/app.ini`_

My own alternative(s):
_`CustomPath`_`/conf/app.ini`
or
`$CustomPath/conf/app.ini`

Reply:
Then the path shouldn't be italic at all to keep consistency

Reply:
If hugo honors HTML we could craft it manually:

`<code><i>CustomPath</i>/conf/app.ini</code>`
<code><i>CustomPath</i>/conf/app.ini</code>

Reply:
It appears by default it does not honor raw HTML and we would need to make a config change for it, which I'm hesitant to do for just a single page.

Considering that, personally I think perhaps the following:
```suggestion
look for its configuration file _`CustomConf`_ in `%(CustomPath)/conf/app.ini`, and will use the
```
`%(CustomPath)/conf/app.ini`
As that matches how we refer to env vars in paths in the FAQ.

Reply:
I think this looks better. Thoughts?
```suggestion
look for its configuration file _`CustomConf`_ in `$(CustomPath)/conf/app.ini`, and will use the
```

Reply:
I thought of that as well, my only reservation was that syntax technically does something else in bash.  (I know this isn't intended to be ran, but figured it might be confusing when presented)

Reply:
I've seen this kind of syntax used elsewhere in other documentations and I don't believe people will get confused but it's your call.

Reply:
Sure, fine by me. Better than what we have now! 😄 
Thanks for the PR!